shelley/chain-and-ledger/executable-spec/cddl-files/shelley.cddl

```diff
@@ -206,7 +206,14 @@ transaction_metadata =
 
 vkeywitness = [ $vkey, $signature ]
 
-bootstrap_witness = [ $vkey, $signature, bytes, bytes, bytes ]
+; explicit CBOR list of length 5
+bootstrap_witness = 
+  [ public_key      : bytes .size 32
```

Comment:
thank you @SebastienGllmt !

the CDDL tests are failing, since they use the mock crypto implementation where the pubkey is 8 bytes. (you can see the definition of `$vkey` in `cddl-files/mock/crypto.cddl`. similarly with `$signature` below.

I think the best thing to do is to make a new file `cddl-files/real/crypto.cddl` and define
```
$vkey /= bytes .size 32
$signature /= bytes .size 64
```

and keep `$vkey` and `$signature` as here in this file.